src/main/resources/static/styles.css

```diff
@@ -0,0 +1,35 @@
+body {
+    background-color: black;
+    background-image: radial-gradient(rgba(0, 150, 0, 0.75), black 120%);
+    height: 100vh;
+    margin: 0;
+    overflow: hidden;
+    padding: 2rem;
+    color: white;
+    font: 1.3rem Inconsolata, monospace;
+    text-shadow: 0 0 5px #C8C8C8;
+
+    &::after {
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100vw;
+        height: 100vh;
+        background: repeating-linear-gradient(0deg,
+                rgba(black, 0.15),
+                rgba(black, 0.15) 1px,
+                transparent 1px,
+                transparent 2px);
+        pointer-events: none;
+    }
+}
+
+::selection {
+    background: #0080FF;
+    text-shadow: none;
+}
+
+pre {
+    margin: 0;
+}
```

Comment:
CSS can do miracles :)

Reply:
Yep, it does. We will play more with it, in the future